StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1024ColonsMustBeSpacedCorrectly.cs

```diff
@@ -138,8 +138,8 @@ private void HandleColonToken(SyntaxTreeAnalysisContext context, SyntaxToken tok
                 }
             }
 
-            bool? hasPrecedingSpace = null;
-            if (!token.HasLeadingTrivia)
+            bool hasPrecedingSpace = token.HasLeadingTrivia;
+            if (!hasPrecedingSpace)
```

Comment:
This code will accept a comment block `/* ... */` as a space. But its not a major issue.